lighthouse-cli/bin.js

```diff
@@ -8,6 +8,19 @@
 const fs = require('fs');
 const path = require('path');
 
+/*
+ * The relationship between these CLI modules:
```

Comment:
should this be in `index.js` since that's likely the place you'd look first, or here, as the "real" beginning